packages/kbn-router-to-openapispec/src/oas_converter/kbn_config_schema/post_process_mutations/index.ts

```diff
@@ -39,6 +39,8 @@ const walkSchema = (ctx: IContext, schema: OpenAPIV3.SchemaObject): void => {
     mutations.processMap(ctx, schema);
   } else if (schema.type === 'string') {
     mutations.processString(schema);
+  } else if ((schema.type as string) === 'stream') {
+    mutations.processStream(schema);
```

Comment:
It's possible that a response body is a stream, in this case we just map it to `object` which is similar to "any" in OAS. Not sure there is good way to represent this at present.